sdk/cosmosdb/cosmos/test/public/functional/globalEndpointManager.spec.ts

```diff
@@ -41,14 +43,45 @@ const databaseAccountBody: any = {
 
 describe("GlobalEndpointManager", function () {
```

Comment:
You may want to assert that the `unavailable` flag toggles between true and false as appropriate in the tests now that it's added back.

Reply:
That's already verified in our tests.